libfabric.spec.in

```diff
@@ -32,9 +32,11 @@ Development files for the libfabric library.
 
 %prep
 %setup -q -n %{name}-%{version}
+wget https://github.com/01org/opa-psm2/archive/PSM2_10.3-37.tar.gz
```

Comment:
Please do not put an unconditional `wget` in the specfile that is used for the whole libfabric package.

Reply:
Thanks @jsquyres. This is just a POC for some quick testing. The approach, if being adopted in the future, will need a lot of fine tuning.